.github/workflows/docker-builds.yml

```diff
@@ -7,6 +7,7 @@ on:
       - .ci/docker/**
       - .github/workflows/docker-builds.yml
       - requirements-lintrunner.txt
+      - examples/arm/**
```

Comment:
@huydhn IIUC, dock image rebuild will be triggered if anything changed in this dir. Should we narrow the triggering condition down to detecting changes in `examples/arm/setup.sh` and `examples/arm/ethos-u-setup/**` eg reference_model version bump? @Jerry-Ge only these two places matters regarding setup the test env right?

Reply:
You're right that Docker rebuild will be triggered for all changes in `examples/arm` folder. I could narrow it down to only these places if only they are needed.  Although, may be it's easier to just do a blanket include here.

Reply:
Hmm, thinking a bit more, I think we should keep the rule as it's because it reflects the way the whole directory is copied to Docker `cp -r ../../examples/arm/ ./arm`.

Reply:
@huydhn Remind me if we do need to copy the entire `examples/arm` folder. IIUC, those [*`.py` files and the `run.sh` script](https://github.com/pytorch/executorch/tree/main/examples/arm) are not needed by the docker image. Those files may be updated more frequently than the actual dependency for docker image build. Maybe a better way is to collect all deps that docker image needs in a single dir eg `examples/arm/setup/**` and monitor the changes there for docker image rebuild?

Reply:
Yeah, putting them altogether in a subdirectory like `examples/arm/setup/**` makes sense to me.  As I'm reorganize these scripts a bit to fix the Docker build part, let me include that part there too